src/components/ButtonLink.tsx

```diff
@@ -85,9 +85,8 @@ const SecondaryScrollLink = styled(StyledScrollButton)`
   }
 `
 
-interface IProps extends ILinkProps {
+interface IProps extends ILinkProps, MarginProps {
   toId?: string
-  isSecondary?: boolean
```

Comment:
Moving `isSecondary` to the base `Link` as the line where it says:
```tsx
 <Link isSecondary to="/developers/">
```
also needed this prop

Reply:
Thank you! Changes reverted based on ^.

Do you want me to include the update of all of this case to use `ButtonLink` instead of `Link` in this PR as well?

Reply:
I saw that you already did that, awesome!

Reply:
Yes, within this PR, the one in `studio.tsx` will be the only case. I was more referring to all the other use cases that should use `ButtonLink` instead of `<Link isSecondary ...`, if we needed to change the rest of them outside of this migration PR